Assets/Scripts/SceneManagement/SavedFileCleaner.cs

```diff
@@ -1,25 +1,6 @@
 using UnityEngine;
-using UnityEditor;
 
-   /// <summary>
-   /// This is only in place to delete the save game file eveny tile the game is stopped being played while testing.
-    /// </summary>
-
-// [InitializeOnLoadAttribute]
 public class SavedFileCleaner : MonoBehaviour
 {
-    // // register an event handler when the class is initialized
-    // static SavedFileCleaner()
-    // {
-    //     EditorApplication.playModeStateChanged += LogPlayModeState;
-    // }
-    // private static void LogPlayModeState(PlayModeStateChange state)
-    // {
-    //     if(state == PlayModeStateChange.ExitingPlayMode)
-    //     {
-    //         Debug.Log("Delete save file");
-    //         EventHandler.CallQuitPlayGameEvent();
-    //     }
-    // }
-
+    // This is here to fix merge conflict.
```

Comment:
?

Reply:
I had changed the name of the file and then when tried to merge it was giving an error. So I just created a new blank file with the same old name to fix the issue. It's on my list to correct.